sbnanaobj/StandardRecord/SRSBNDCRTVeto.h

```diff
@@ -0,0 +1,64 @@
+////////////////////////////////////////////////////////////////////////
+/**
+ * @file   sbnanaobj/StandardRecord/SRSBNDCRTVeto.h
+ * @brief  Definition of event veto from CRT information.
+ * @author Alexander Antonakis
+ * @email: aantonakis@ucsb.edu
```

Comment:
Doxygen has [no command for e-mail by itself](https://www.doxygen.nl/manual/commands.html), but it will automatically render as a link an e-mail address anywhere:

```suggestion
 * @author Alexander Antonakis (aantonakis@ucsb.edu)
```